src/thrift-client/TgtInterfaceImpl.cpp

```diff
@@ -760,6 +802,7 @@ std::pair<Request*, bool> StordVmdk::NewRequest(Request::Type type,
 	}
 
 	++stats_.pending_;
+	++stord_stats_.pending_;
```

Comment:
this seems redundant

Reply:
stats_ is VMDK instance specific, while stord_stats_ is static across VMDK.